includes/discovery/os/nos.inc.php

```diff
@@ -1,5 +1,13 @@
 <?php
 
+$fabos_objectid = array(
```

Comment:
Good find, I modified this from my initial thought that this was FABOS when it turned out to be NOS.